Add known issues section for Kiali (#364)
kyma-project/website: content/blog-posts/2019-06-13-release-notes-1.2/index.md

- [Observability](#observability) - Early version of Kiali added to Istio
- [Service Mesh](#service-mesh) - Istio update to version 1.1.6

> Read about a known issue for [Observability](###known-issues).

Reviewer: I think that triple `#` us unnecessary and link doesn't work :) 
```markdown
Read about a known issue for [Observability](#known-issues).
```

Author: changed

> Read about a known issue for [Observability](###known-issues).

Reviewer 2: ```suggestion
Read about a known issue for [Observability](#known-issues).
```

Early integration of Kiali is available as part of Istio. To enable Kiali, ensure that the monitoring module is installed and set the **kiali.enabled** parameter to `true`. The Kiali UI will be accessible under the `kiali` subdomain. The early integration is based on static user security. To learn how to get the Kiali UI password, see [this](https://github.com/kyma-project/kyma/tree/master/resources/istio) document.

### Known issues

> Kiali is not accessible after installation when enabled as a part of the installer configuration. As an example, for minikube it should be accessible at https://kiali.kyma.local.

Reviewer 2: ```suggestion
Kiali is not accessible after installation when enabled as part of the Installer configuration. It should be accessible on Minikube at `https://kiali.kyma.local`.
```

> As a workaround prefix the hosts attribute in kiali-virtualservice resource with "kiali":

Reviewer 2: ```suggestion
To access it this way, use the workaround for now and add the `kiali` prefix to the **hosts** attribute in the `kiali-virtualservice` resource:
```

```
kubectl -n istio-system edit virtualservices kiali-virtualservice
```
```

>   hosts:

Reviewer: Please remove white spaces

>   - kiali.kyma.local

Reviewer: Please remove white spaces
